File: Two_sum/two_sum.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int i = 0;
        int j = 1;
        int x, y;
        int end = numbers.length - 1;
        int[] twoIndex = new int[2];
        
        while(i <= end)
        {
            if(numbers[i] + numbers[j] == target)
            {
                x = i + 1;
                y = j + 1;
                twoIndex[0] = x;
                twoIndex[1] = y;
                i++;
                j++;
                return twoIndex;
            }
            else if(j == end)
            {
                i++;
                j = i + 1;
            }
            else{
                j++;
            }
        }
        return twoIndex;
    }
}

File: Two_sum/two_sum.cs
public class Solution {
    public int[] TwoSum(int[] numbers, int target) {
        int i = 0;
        int j = 1;
        int x, y;
        start = 0;
        int end = numbers.Length - 1;
        int middle = 0;
        int[] twoIndex = new int[2];
        while(start <= end)
        {
            middle = (start + end) / 2;
            if(numbers[middle] > target)
            {
                end = middle - 1;
            }
            else if(numbers[middle] <= target)
            {
                
            }
        }
    }
}